crawler/src/main/java/com/movie/recommender/crawler/DB.java

```diff
@@ -0,0 +1,57 @@
+package com.movie.recommender.crawler;
+
+import com.google.gson.Gson;
+import com.mongodb.client.MongoClient;
+import com.mongodb.ConnectionString;
+import com.mongodb.client.MongoClients;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+import org.bson.Document;
+import org.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+public class DB {
```

Comment:
remove the database class, as it is currently out of scope for this implementation